src/reducers/index.js

```diff
@@ -0,0 +1,10 @@
+import { combineReducers } from 'redux';
+import booksReducer from './books';
+import filter from './filter';
+
+const combinedReducers = combineReducers({
+  books: booksReducer,
+  filter,
```

Comment:
**OPTIONAL**: Consistency makes the code more readable/maintainable. You are naming one reducer `booksReducer` and the other `filter`. Choose one form or the other but try to be consistent about your naming.